### ISSUE TITLE
Missing dev files?

### ISSUE THREAD

**Issue by mcepl** [ikalnitsky/pythonista#14]
Hi,

I am trying to use your Docker image for testing [M2Crypto](https://gitlab.com/mcepl/m2crypto/builds/1103018) with python 2.6 and it seems to me that your are missing files packaged in Debian in ``-dev`` (or ``-devel`` in CentOS-speak) package. Would it be possible to add them?

Thank you

**Comment by ikalnitsky** [comment]
@mcepl Yep, sounds like a bug. Thanks for reporting! I'll check it out.
